microsoft-edge/web-platform/how-to-detect-win11.md

```diff
@@ -82,6 +82,42 @@ navigator.userAgentData.getHighEntropyValues(["platformVersion"])
 
 ```
 
+<!-- ====================================================================== -->
+## Sample code for detecting ARM or x86
+
+Use detection of CPU architecture to have your website automatically download the version of your app that's built specifically for the user's CPU.
+
+CPU detection is particularly helpful for ARM-based devices, so that a customer using an ARM device automatically downloads the native ARM version of an application.  This prevents the user from inadvertently installing an app that's built for x86, and then experiencing reduced performance due to emulation.
```

Comment:
```suggestion
CPU detection is particularly helpful for ARM-based devices, so that a customer using an ARM device automatically downloads the native ARM version of the application.  This prevents the user from inadvertently installing an app that's built for x86, and then experiencing reduced performance due to emulation.
```